integration-tests/camel/rpkgtests/pom.xml

```diff
@@ -324,6 +324,11 @@
             <artifactId>camel-quarkus-integration-test-xstream</artifactId>
             <version>${camel-quarkus.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.debezium</groupId>
+            <artifactId>debezium-quarkus-outbox-integration-tests</artifactId>
+            <version>${debezium-outbox-quarkus.version}</version>
+        </dependency>
```

Comment:
I like that it's not nested under Camel, except for this change.

Reply:
Ah, good catch as I missed that during rebase.  I assume this should therefore be under the current `other\rpkgtests\pom.xml` ?

Reply:
Would you mind moving those two modules out of `other` and remove it? That way you could add this specific change to `rpkgtests` and add your `debezium` IT module.

Reply:
Not at all @aloubyansky, I did this as a separate commit under 0dd59bf.

Reply:
Great, thanks a lot! This specific change does not appear to be removed though.

Reply:
I thought you'd move it to integration-tests/rpkgtests/pom.xml.

Reply:
Oops, I had thought I had done that previously.  Fixed in c661e64.